lib/SQLiteStorage.js

```diff
@@ -0,0 +1,159 @@
+const StorageInterface = require('./StorageInterface')
+
+class SQLiteStorage extends StorageInterface {
+    constructor(config) {
+        super()
+        this.adapter = null
+        this.config = config
+    }
+
+    async _getAdapter() {
+        if (!this.adapter) {
+            // Create a simple SQLite adapter using better-sqlite3
+            const Database = require('better-sqlite3')
+            const path = require('path')
+            
+            this.adapter = {
+                readFile: async (filePath) => {
+                    // Return empty array for raw data files - SQLite doesn't store these
+                    // The bot will just start fresh without previous raw data
+                    return JSON.stringify([])
+                },
+                writeFile: async (filePath, data) => {
+                    // All data goes to SQLite - no JSON files created
+                    const pathParts = filePath.split('/')
+                    const fileName = pathParts[pathParts.length - 1]

```

Comment:
how does this work for other mines that you didn't add explicit `if`s for? 
at first read it seems like it would only handle  `Candles`, `Volumes`, and `OHLCVs` 

probably missing something but it isn't obvious how other data mines / indicators would get tables created and populated in a migration